src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1

```diff
@@ -534,6 +535,93 @@ class Project: ADOSVTBase
         return $controlResult
     }
 
+    hidden [ControlResult] CheckMaxBACount([ControlResult] $controlResult){
+        $controlResult.VerificationResult = [verificationResult]::Failed;
+        $TotalBAMembers = 0;
+        if ($this.BAMembers.Count -eq 0) {
+            $this.BAMembers += @([AdministratorHelper]::GetTotalBAMembers($this.OrganizationContext.OrganizationName,$this.ResourceContext.ResourceName))
+        }
+        if($this.BAMembers.Count -gt 0)
+        {
+            $TotalBAMembers = $this.BAMembers.Count
+            $controlResult.AddMessage("There are a total of $TotalBAMembers Build Administrators in your project.")
+            $controlResult.SetStateData("Count of Build Administrators: ",$TotalBAMembers)
+        }
+
+
+        if ($TotalBAMembers -gt 0)
```

Comment:
Already has if at line 544,  if($this.BAMembers.Count -gt 0) cant we add this if code udeer same